CONTRIBUTING.rst

```diff
@@ -618,11 +640,21 @@ Here's a more detailed flowchart of how to submit changes.
      protocol.
 
      .. _SSH key: https://help.github.com/articles/connecting-to-github-with-ssh/
-
-#. Issue a pull request at
-   https://github.com/yt-project/yt/pull/new/master
-   A pull request is essentially just asking people to review and accept the
+#. Issue a pull request at https://github.com/yt-project/yt/pull/new/master A
+   pull request is essentially just asking people to review and accept the
    modifications you have made to your personal version of the code.
+#. Once your pull request is merged, sync up with the main yt repository by
+   pulling from the ``upstream`` remote::
+
+     git checkout master
+     git pull upstream master
+     git branch -d my-first-pull-request
+     git push origin --delete my-first-pull-request
+
+The last two commands will delete your branch name locally and on your
```

Comment:
I guess the confusion might come from the question "which remote?" and you're saying "on your fork" to mean that's really it's `origin`, so I get it... But I still think that "your fork" does not necessarily mean its remote version. Maybe "on your remote" then?